compiler/src/iree/compiler/Codegen/Interfaces/PartitionableLoopsInterface.cpp

```diff
@@ -125,19 +129,20 @@ struct FftOpPartitionableLoops
     : public PartitionableLoopsInterface::ExternalModel<
           FftOpPartitionableLoops, IREE::LinalgExt::FftOp> {
   llvm::SmallVector<unsigned> getPartitionableLoops(
-      Operation *op, unsigned maxNumPartitionedLoops) const {
+      Operation *op, llvm::Optional<unsigned> maxNumPartitionedLoops) const {
     auto fftOp = cast<IREE::LinalgExt::FftOp>(op);
     auto range = llvm::seq<unsigned>(0, fftOp.getOperandRank());
     SmallVector<unsigned> partitionableLoops(range.begin(), range.end());
     // Indices matter for coeff computation.
     if (!fftOp.hasCoeff()) {
       partitionableLoops.pop_back();
     }
-    if (partitionableLoops.size() > maxNumPartitionedLoops) {
-      partitionableLoops.erase(
-          partitionableLoops.begin(),
-          std::next(partitionableLoops.begin(),
-                    partitionableLoops.size() - maxNumPartitionedLoops));
+    if (maxNumPartitionedLoops.has_value() &&
```

Comment:
Same here . Style nit. Just the indentations gotten a bit unreadable. Maybe flip the condition, so
```
if (!maxNumPartitionableLoops.has_value() || partitionableLoops.size() <= maxNumPartitionedLoops.value()) {
  return partitionableLoops;
}
partitionableLoops.erase(...)

```